sycl/doc/extensions/DataFlowPipes/data_flow_pipes_rev4_proposed.asciidoc

```diff
@@ -170,20 +170,52 @@ The pipe class exposes static member functions for writing a data word to a pipe
 
 Blocking and non-blocking forms of the read and write members are defined, with the form chosen based on overload resolution.
 
+In the experimental API version, read/write methods take template arguments, which can contain the latency control properties `latency_anchor_id` and/or `latency_constraint`.
```

Comment:
This extension is a little weird now because only some of the APIs are experimental.  I think this document should make it more clear about which parts are experimental, and it also needs to include wording stating that experimental APIs could change in future version of the extension without maintaining backward compatibility.

I'd suggest reverting the section "Read/write member functions, and pipe template parameters" to its original form and only describe the non-experimental APIs (the ones without the template parameters).  Then I'd add a new section that is devoted to the experimental APIs.  That section should start with a prominent NOTE saying something like:

> NOTE: The APIs described in this section are experimental.  Future versions of this extension may change these APIs in ways that are incompatible with the versions described here.

Reply:
I agree that will make the structure clearer. I'll do the change.